src/doghouse/adapters/github/gh_cli_adapter.py

```
@@ -0,0 +1,176 @@
+
+class GhCliAdapter(GitHubPort):
+    """Adapter for GitHub using the 'gh' CLI."""
+    
+    def __init__(self, repo_owner: Optional[str] = None, repo_name: Optional[str] = None):
+        self.repo_owner = repo_owner
+        self.repo_name = repo_name
+        self.repo = f"{repo_owner}/{repo_name}" if repo_owner and repo_name else None
+
+    def _run_gh(self, args: List[str]) -> str:
+        """Execute a 'gh' command and return stdout."""
+        cmd = ["gh"] + args
```

Ruff (0.15.7)

[warning] Consider `["gh", *args]` instead of concatenation

Replace with `["gh", *args]`

(RUF005)